fairlearn/adversarial/_adversarial_mitigation.py

```diff
@@ -735,9 +739,11 @@ def _set_predictor_function(self):
         elif isinstance(self.predictor_function_, str):
             kw = self.predictor_function_
             if kw == "binary":
-                self.predictor_function_ = lambda pred: (pred >= self.threshold_value).astype(
-                    float
-                )
+
+                def binarization(pred):
```

Comment:
is this used anywhere?

Reply:
I believe there was a test failing, but it has been now many weeks since I wrote this and started this branch and I am not 100% sure :S I will review it all once again.